.envrc

```diff
@@ -1,8 +1,7 @@
 ### General env vars ###
 export WORKSPACE_ROOT=$(pwd)
 export RUST_LOG_FORMAT=devel
-export RUST_BACKTRACE=1
-export RUST_LOG=query_engine_tests=trace,query_engine=debug,query_core=debug,query_connector=debug,sql_query_connector=debug,prisma_models=debug,engineer=info,sql_introspection_connector=debug,tiberius=trace,quaint=debug,mongodb_query_connector=trace
+export RUST_LOG=info
```

Comment:
This is mostly for the client team: would opting in to more verbose logs in some places manually not make more sense?